Quotes for MCOfficer's person ship (#5747)
endless-sky/endless-sky: data/persons.txt

	phrase
		phrase
			"friendly author"
	phrase
		word
			"Remember, overkill is underrated."
			"When in doubt, add more flammenwerfer."
			"Why settle for something reasonable, when you can have something completely bonkers?"
			"Bactrian Gaming."
			"His Union's Qi Pianist."
			"I'm basically a computer gynecologist."
			"Your ship looks like it came out of a liquidizer!"
			"Look at me. I am the Captain now."

> 			"Pronounce 'Wrzlprnft'. You have 5 seconds."

Reviewer: ```suggestion
			`Pronounce "Wrzlprnft." You have 5 seconds.`
```

			"Everything you see here has been created by volunteers. You can join us if you want!"

			"To this day, I can still hear the Teeworlds title music."
			"I am a light sleeper; but I am a heavy dreamer." # Owl City
			"Did you know Viggo broke his toe?" # LoTR
			"Meet me at the Prancing Pony." # LoTR
			"This isn't my first ship. I fondly remember The Blin Machine and The Chernomobil." # Life of Boris
			"Heard of solar panels? Now you can eat sunlight!" # bill wurtz

> 			"So you want to be a pirate, m'lad? Well, let me tell ye: Being a pirate, be all about Branding." # CGP Grey

Reviewer: ```suggestion
			"So you want to be a pirate, m'lad? Well, let me tell ye: being a pirate, be all about Branding." # CGP Grey
```

			"Do not greed for the Thrumbo horns." # RimWorld

			# Walter Moers: The Mugg's Code of Conduct
			"Honour thy muggroom."
			"If thou seest two sticks lying one on top of the other, thou shalt walk backwards over them with thy left foot first, not with thy right. Moreover, thou shalt not devour them."
			"Should a vulture's shadow fall across a fire that has gone out, thou shalt rekindle it three times or a great misfortune will ensue."
			"If thou cross the path of a white cockerel seated on two superimposed sticks, thou shalt not strike it, nor shalt thou address it by name nor partake of the said sticks."
			"Thou shalt not finkle backwards."
			"Thou shalt not finkle forwards."

			# SovietWomble
			"Get the doomsday rounds."

> 			"I wouldn't recommend shooting at me, because your gun goes 'pew pew', but my gun goes BBBRRRRRRR!"

Reviewer: ```suggestion
			`I wouldn't recommend shooting at me, because your gun goes "pew pew," but my gun goes BBBRRRRRRR!`
```

	phrase
		phrase
			"friendly author"
	phrase
		word
			"Remember, overkill is underrated."
			"When in doubt, add more flammenwerfer."
			"Why settle for something reasonable, when you can have something completely bonkers?"
			"Bactrian Gaming."
			"His Union's Qi Pianist."
			"I'm basically a computer gynecologist."
			"Your ship looks like it came out of a liquidizer!"
			"Look at me. I am the Captain now."
			`Pronounce "Wrzlprnft." You have 5 seconds.`
			"Everything you see here has been created by volunteers. You can join us if you want!"

			"To this day, I can still hear the Teeworlds title music."
			"I am a light sleeper; but I am a heavy dreamer." # Owl City
			"Did you know Viggo broke his toe?" # LoTR
			"Meet me at the Prancing Pony." # LoTR
			"This isn't my first ship. I fondly remember The Blin Machine and The Chernomobil." # Life of Boris
			"Heard of solar panels? Now you can eat sunlight!" # bill wurtz

> 			"So you want to be a pirate, m'lad? Well, let me tell ye: being a pirate, be all about Branding." # CGP Grey

Reviewer: ```suggestion
			"So you want to be a pirate, m'lad? Well, let me tell ye: being a pirate, be all about Branding." # CGPGrey
```

			"Do not greed for the Thrumbo horns." # RimWorld

			# Walter Moers: The Mugg's Code of Conduct

> 			"Honour thy muggroom."

Reviewer: ```suggestion
			"Honor thy muggroom."
```
